.github/workflows/test.yml

```diff
@@ -0,0 +1,96 @@
+name: Test Build
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  test:
+    name: Gatsby + Craft with PHP ${{ matrix.php-versions }} + ${{ matrix.db }}
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
+        php-versions: ["7.4"]
+        db: ["mysql", "pgsql"]
+    services:
```

Comment:
@mattstein the spacing here might make more sense visually to add a new line before the "section" (e.g. a new line before "services" and make the rest of the file consistent - the steps are not grouped.

Reply:
Done.